src/UnitfulRecipes.jl

```diff
@@ -49,6 +49,9 @@ function fixaxis!(attr, x, axisletter)
     if (axisletter == :y) && haskey(attr, :ribbon)
         ustripattribute!(attr, :ribbon, u)
     end
+    if (axisletter == :y) && haskey(attr, :fillrange)
+        ustripattribute!(attr, :fillrange, u)
+    end
```

Comment:
Maybe this is cleaner?

```suggestion
    if (axisletter == :y)
        haskey(attr, :ribbon) && ustripattribute!(attr, :ribbon, u)
        haskey(attr, :fillrange) && ustripattribute!(attr, :fillrange, u)
    end
```

Reply:
I think `ustripattribute!` itself handles the case of missing key. So the `haskey` checks here can be removed?

Reply:
Ah you're probably right here I guess:

https://github.com/jw3126/UnitfulRecipes.jl/blob/465510a11307fbf0c04168f611b6607b34c5dfa4/src/UnitfulRecipes.jl#L137-L146

I wonder why we had that in the first place for `:ribbon`...

Reply:
abundance of caution :smile:

Reply:
So maybe just

```suggestion
    if (axisletter == :y)
        ustripattribute!(attr, :ribbon, u)
        ustripattribute!(attr, :fillrange, u)
    end
```

?